packages/scripts/test/application.unit.ts

```diff
@@ -133,6 +139,60 @@ describe('Application', function() {
                 expect(await getBodyContent(page)).to.equal('from server');
             });
         });
+
+        it('hot reloads config files', async () => {
+            const modifiedConfigValue = 'modified config';
+            const originalConfigValue = 'original config';
+            const configFilePathInRepo = fs.join(useConfigsFeaturePath, 'feature', 'example.config.ts');
+
+            // creating config file
+            await createConfigFile(
+                configFilePathInRepo,
+                `
+import UseConfigs from './use-configs.feature';
```

Comment:
extract this string to a function accepting a value and returning the text (probably uses a template string).
then, use the function below, when changing the file. remove the readFile. avoid replace call. it's just noise.